corehq/apps/sms/api.py

```diff
@@ -41,6 +41,10 @@ class BackendAuthorizationException(Exception):
     pass
 
 
+def get_utcnow():
```

Comment:
yeah, the only reason i added this was so i could mock the current timestamp in the tests. it's not possible to patch datetime.datetime directly (see http://stackoverflow.com/questions/192649/can-you-monkey-patch-methods-on-core-types-in-python/192857#192857)